container/nextcloud/Dockerfile

```diff
@@ -0,0 +1,85 @@
+FROM archlinux:latest
+MAINTAINER Sebastian Selka
+
+RUN pacman --noconfirm -Syu
+RUN pacman --noconfirm -S \
+        supervisor \
+        nginx-mainline \
+        wget \
+        php-legacy-fpm \
+        php-legacy-pgsql \
+        php-legacy-apcu \
+        php-legacy-redis \
+        php-legacy-gd \
+        nextcloud \
+        fcron
+
+EXPOSE 80 80
+
+RUN cd /usr/share/webapps/nextcloud/apps/ && \
+    wget -O calendar.tar.gz https://github.com/nextcloud-releases/calendar/releases/download/v4.7.11/calendar-v4.7.11.tar.gz && \
```

Comment:
`ADD` kann auch `xyz.tar.gz` für dich runterladen und entpacken

Reply:
Das geht leider nicht mehr :/ Das war mal kurz in Docker drin. Das wurde wieder ausgebaut :/